src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,84 @@
-export const Pagination = () => {};
+import { FC } from 'react';
+import cn from 'classnames';
+import { PaginationItem } from '../PaginationItem';
+import { getNumbers } from '../../utils';
+
+interface Props {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+  setCurrentPage: (page: number) => void;
+}
+
+export const Pagination: FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+  setCurrentPage,
```

Comment:
Looks like you dont need setCurrentPage prop. You can do its logic using the onPageChange